src/distbin-html/an-activity.ts

```diff
@@ -586,6 +595,12 @@ export const createActivityCss = () => {
     .action-show-raw pre {
       color: initial
     }
+    code {
+      background-color: rgba(0, 0, 0, 0.05);
+      display: inline-block;
+      width: 100%;
```

Comment:
Please remove the `width: 100%`.

`<code />` can be inline in text.

e.g. try testin gwith both inline and block-level codes by typing this in the editor and posting:

<pre>
test:

```javascript
const test = 1
```

works?

how about `const test2 = 1`. Does that work inline?
</pre>

Reply:
Yes, that's right, I didn't thought about inline code